### PR TITLE
Prepare 0.27 release

### PR DESCRIPTION
**Pull request by stackoverflow**

### CODE REVIEW

File: DEVELOPMENT.adoc [hunk 1]
```diff
@@ -50,8 +50,9 @@ To debug the plugin, go to the _runIde_ task in the Gradle window and debug it.
 
 Before a release, run Code->Inspect Code->Whole Project (at least) for pkl-hub project and check the results.
 
-The release CI build publishes the plugin zip but doesn't update the plugin update site.
-To update the plugin update site, edit `updatePlugins.xml` on gh-pages branch.
+The release CI build publishes the plugin zip and the `updatePlugins.xml`. +
+The plugin update site always points to the latest released `updatePlugins.xml`, so nothing
+has to be done there.
```
Comment (holzensp) [inline DEVELOPMENT.adoc:55]: The negative instruction ("don't do anything") only makes sense in the context of a previous positive instruction. Maybe leave out altogether, or
```suggestion
The plugin update site is automatically updated.
```

**Review by holzensp — APPROVED** [review]
LGTM

File: DEVELOPMENT.adoc (hunk 1 above)
Comment (stackoverflow) [reply DEVELOPMENT.adoc]: Makes sense.

**Review by bioball — APPROVED** [review]
LGTM

**Pull request merged**